api/config/config.go

```diff
@@ -124,6 +124,10 @@ type DatabaseConfig struct {
 	Password      string `envconfig:"DATABASE_PASSWORD" required:"true"`
 	Database      string `envconfig:"DATABASE_NAME" default:"mlp"`
 	MigrationPath string `envconfig:"DATABASE_MIGRATIONS_PATH" default:"file://db-migrations"`
+
+	ConnMaxIdleTime time.Duration `envconfig:"DATABASE_CONN_MAX_IDLE_TIME" default:"10m"`
+	MaxIdleConns    int           `envconfig:"DATABASE_MAX_IDLE_CONNS" default:"10"`
+	MaxOpenConns    int           `envconfig:"DATABASE_MAX_OPEN_CONNS" default:"100"`
```

Comment:
Do we need to update helm chart to include these?

Reply:
Updated